support-frontend/package.json

```diff
@@ -11,7 +11,7 @@
     "build-ssr": "webpack --config webpack.ssr.js && node scripts/ssrScript.js",
     "build-prod": "npm-run-all clean validate test storybook-static 'webpack --config webpack.prod.js --env.prod'",
     "webpack": "webpack",
-    "webpack-dev-server": "webpack-dev-server --config webpack.dev.js --port 9211 --public support.thegulocal.com",
+    "webpack-dev-server": "webpack-dev-server --config webpack.dev.js --port 9211 --public support.thegulocal.com --disable-host-check",
```

Comment:
is it possible to add a comment with a link to the issue jonny was looking at?

Reply:
@johnduffell comments are not supported in json files so we settled for a PR with the documentation 😢